gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc_asyncio.py.j2

```diff
@@ -225,13 +228,13 @@ class {{ service.grpc_asyncio_transport_name }}({{ service.name }}Transport):
         client.
         """
         # Sanity check: Only create a new client if we do not already have one.
-        if 'operations_client' not in self.__dict__:
-            self.__dict__['operations_client'] = operations_v1.OperationsAsyncClient(
+        if self._operations_client is None:
```

Comment:
Why are you changing this code (looks unrelated to http transport stuff)?

Reply:
https://github.com/googleapis/gapic-generator-python/pull/688#discussion_r516194162
I decided to update the other templates as well for consistency.

Reply:
+1 for consistency